src/nim_status_client.nim

```diff
@@ -219,7 +217,6 @@ proc mainProc() =
 
   statusq_registerQmlTypes()
 
-  app.installEventFilter(osThemeEvent)
   app.installEventFilter(urlSchemeEvent)
```

Comment:
Isn't the urlSchemeEvent removed as well?

Reply:
Yeah it should be but it's not that easy :) See below